PowerPointLabs/PowerPointLabs/Ribbon1.cs

```diff
@@ -152,17 +155,45 @@ public void ToggleVisibility(Office.IRibbonControl control, bool pressed)
             _ribbon.InvalidateControl("VisibleFormatShapes");
         }
 
+        // Toggles the boolean controlling whether images should be compressed.
+        public void ToggleImageCompression(Office.IRibbonControl control, bool pressed)
+        {
+            ShouldCompressImages = pressed;
+            _ribbon.InvalidateControl("ShouldCompressImagesCheckbox");
+
+            if (ShouldCompressImages)
+            {
+                GraphicsUtil.PictureExportingRatio = 96.0f / 72.0f;

```

Comment:
it will be better if these ratio values will be stored in GraphicsUtil as a constant so that if anyone wants to find these values then they will directly go to GraphicsUtil, not Ribbon1.

Reply:
Done! moved them into GraphicsUtil